src/Libraries/DSCPython/DSCPythonCodeCompletionProviderCore.cs

```diff
@@ -95,14 +104,24 @@ private object ExecutePythonScriptCode(string code)
                 return items;
             }
 
+            var isCLRType = d.GetPythonType()?.ToString().Contains("CLR") ?? false;
```

Comment:
@pinzart90 this is kind of ugly, in our wrapper of PythonNet we added `GetManagedObject` does that work?
https://git.autodesk.com/Dynamo/pythonnet/blob/dynamo_master/src/runtime/pyobject.cs#L1444

Reply:
Sadly GetManagedObject does not work unless we call it on an object evaluateed as clr.GetClrType(type)
I did however refactor the code a bit so that searching for the "CLR" keyword only happens in limited cases (only CLR namespaces ...so far)